state/cleanup_test.go

```diff
@@ -395,9 +395,9 @@ func (s *CleanupSuite) TestCleanupForceDestroyedControllerMachine(c *gc.C) {
 	s.assertCleanupRuns(c)
 	c.Assert(machine.SetHasVote(false), jc.ErrorIsNil)
 	// However, if we remove the vote, it can be cleaned up.
-	// ForceDestroy sets up a cleanupForceDestroyedMachine, which calls EnsureDead which sets up a cleanupDyingMachine
-	// so it takes 2 cleanup runs to run clear
-	s.assertCleanupCount(c, 2)
+	// ForceDestroy sets up a cleanupForceDestroyedMachine, which calls EnsureDead which sets up a cleanupDyingMachine, which in turn creates a delayed cleanupForceRemoveMachine.
```

Comment:
EnsureDead call has been removed... Looks like this comment needs updating.

Reply:
Yeah, it's changed to a call to `advanceLifecycle` so that I could pass maxWait down. Updated.